src/schema/__tests__/city.test.ts

```diff
@@ -166,6 +172,30 @@ describe("City", () => {
       expect(gravityOptions).toMatchObject({ discoverable: true })
       expect(gravityOptions).not.toHaveProperty("displayable")
     })
+
+    it("can request all shows [that match other filter parameters]", async () => {
+      allViaLoader.mockImplementation(() => Promise.resolve(mockShows))
+
+      query = gql`
+        {
+          city(slug: "sacramende-ca-usa") {
+            shows(first: ${MAX_GRAPHQL_INT}) {
```

Comment:
@alloy so the idea is we'll pass in a really large number for `first` here on the client to get all shows?

Reply:
Yep. Not just _a_ large number, though, the max int that the GraphQL spec allows for. Max ints are often used as sentinels to e.g. mean ‘everything’. I definitely did not want to add a parameter (such as `all`) because removing it in the future would be breaking and it may suggest that this is a strategy we approve of in general.